From 99b0ab762acc46d6d527fdc105528f09fb585c48 Mon Sep 17 00:00:00 2001
From: Gerald Combs <gerald@wireshark.org>
Date: Tue, 29 Oct 2013 18:46:49 +0000
Subject: [PATCH 2/4] Copy over r52154 by hand:

  ------------------------------------------------------------------------
  r52154 | etxrab | 2013-09-20 07:19:31 -0700 (Fri, 20 Sep 2013) | 1 line
  Changed paths:
     M /trunk/asn1/nbap/nbap.cnf
     M /trunk/asn1/nbap/packet-nbap-template.c
     M /trunk/epan/dissectors/packet-nbap.c

  DCH-ID can be 255
  ------------------------------------------------------------------------

Update the release notes.

svn path=/trunk-1.8/; revision=52958
---
 asn1/nbap/nbap.cnf               |    2 +-
 asn1/nbap/packet-nbap-template.c |    2 +-
 epan/dissectors/packet-nbap.c    |    4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/asn1/nbap/nbap.cnf b/asn1/nbap/nbap.cnf
index 5c25a4f..44127b4 100644
--- a/asn1/nbap/nbap.cnf
+++ b/asn1/nbap/nbap.cnf
@@ -625,7 +625,7 @@ guint32 payload_crc_value;
 #.FN_BODY DCH-Specific-FDD-Item/dCH-ID
 
   offset = dissect_per_constrained_integer(tvb, offset, actx, tree, hf_index,
-                                           0U, maxNrOfDCHs, &dch_id, FALSE);
+                                                            0U, 255U, &dch_id, FALSE);
 
 	if(g_num_dch_in_flow>0){
 		g_dchs_in_flow_list[g_num_dch_in_flow-1]=dch_id;
diff --git a/asn1/nbap/packet-nbap-template.c b/asn1/nbap/packet-nbap-template.c
index 8d50951..517b1e8 100644
--- a/asn1/nbap/packet-nbap-template.c
+++ b/asn1/nbap/packet-nbap-template.c
@@ -114,7 +114,7 @@ typedef struct
 
 }nbap_dch_channel_info_t;
 
-nbap_dch_channel_info_t nbap_dch_chnl_info[maxNrOfDCHs];
+nbap_dch_channel_info_t nbap_dch_chnl_info[256];
 
 /* Struct to collect E-DCH data in a packet 
  * As the address data comes before the ddi entries
diff --git a/epan/dissectors/packet-nbap.c b/epan/dissectors/packet-nbap.c
index 40eebb1..49fe92c 100644
--- a/epan/dissectors/packet-nbap.c
+++ b/epan/dissectors/packet-nbap.c
@@ -6507,7 +6507,7 @@ typedef struct
 
 }nbap_dch_channel_info_t;
 
-nbap_dch_channel_info_t nbap_dch_chnl_info[maxNrOfDCHs];
+nbap_dch_channel_info_t nbap_dch_chnl_info[256];
 
 /* Struct to collect E-DCH data in a packet 
  * As the address data comes before the ddi entries
@@ -14566,7 +14566,7 @@ dissect_nbap_T_dCH_ID(tvbuff_t *tvb _U_, int offset _U_, asn1_ctx_t *actx _U_, p
 #line 626 "../../asn1/nbap/nbap.cnf"
 
   offset = dissect_per_constrained_integer(tvb, offset, actx, tree, hf_index,
-                                           0U, maxNrOfDCHs, &dch_id, FALSE);
+                                                            0U, 255U, &dch_id, FALSE);
 
 	if(g_num_dch_in_flow>0){
 		g_dchs_in_flow_list[g_num_dch_in_flow-1]=dch_id;
-- 
1.7.10.4

